clients/wallet/src/stellar_wallet.rs

```diff
@@ -305,18 +276,59 @@ impl StellarWallet {
 		Ok(envelope)
 	}
 
+	pub async fn send_payment_to_address_with_extra_operations(
+		&mut self,
+		destination_address: PublicKey,
+		asset: Asset,
+		stroop_amount: StellarStroops,
+		request_id: [u8; 32],
+		stroop_fee_per_operation: u32,
+		extra_operations: Vec<Operation>,
+	) -> Result<TransactionResponse, Error> {
```

Comment:
Actually, the three operations we need, ie. 'payment' vs 'createaccount' vs 'createclaimablebalance' should be used mutually exclusive. Each of these operations is able to hold a specified amount (which will be our redeemed amount). Thus, we don't need to use a combination of them. 

This means, that we actually wouldn't need this function since we don't want to bundle these operations together in one transaction.

Reply:
method removed :bow: